CodingBasics.csproj

```diff
@@ -8,6 +8,12 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Data.SqlClient" Version="5.1.5" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="8.0.2" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="8.0.2" />
```

Comment:
I can see that you're trying to add EntityFramework, but I couldn't find any related methods in the other classes. As a best practice, it's recommended not to reference packages that you're not going to use. Good attempt, but if you realize that you're not going to use this package, it's best to remove it.